apps_wishlist.md

```diff
@@ -39,7 +39,6 @@ The following list is a compiled wishlist of applications that would be nice-to-
 - [ERPnext](https://erpnext.com/download) / [github](https://github.com/frappe/erpnext)
 - [EteSync](https://www.etesync.com/) / [github](https://github.com/etesync)
 - [Epicyon](https://epicyon.net/) / [gitlab](https://gitlab.com/bashrc2/epicyon)
-- [erine.email](https://erine.email/) / [gitlab](https://gitlab.com/mdavranche/erine.email)
```

Comment:
Dunno how you did the merging but this got removed in the process :s

Reply:
indeed ! somthing went wrong in my process... I try to fix it